app-web/src/utils/helpers.js

```diff
@@ -233,6 +233,31 @@ export const isFilterLonely = filters => {
   return numAreFilterable === 1;
 };
 
+export const pillFilter = (filterName, activeFilter) => {
```

Comment:
this function should be pure, it needs to return a new version of the active filters array.